log/development.log

```diff
@@ -5,3 +5,490 @@
   [1m[35m (0.1ms)[0m  [1m[35mCOMMIT[0m
   [1m[35m (0.1ms)[0m  [1m[34mSELECT pg_advisory_unlock(1580829076522303230)[0m
   [1m[35m (0.1ms)[0m  [1m[34mSELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC[0m
+  [1m[35m (0.1ms)[0m  [1m[34mSELECT pg_try_advisory_lock(1580829076522303230)[0m
```

Comment:
Why? 
![](https://media.giphy.com/media/rhYsUMhhd6yA0/giphy-tumblr.gif)

Reply:
it is in the gitignore, i dont know why it is still pushing !